packages/generated-files-bot/src/generated-files-bot.ts

```diff
@@ -206,10 +206,10 @@ export function handler(app: Probot) {
     const owner = context.payload.repository.owner.login;
     const repo = context.payload.repository.name;
     const pullNumber = context.payload.pull_request.number;
-    const prAuthor = context.payload.pull_request.user.login;
+    const sender = context.payload.sender.login;
 
     // ignore PRs from a configurable list of authors
-    if (config.ignoreAuthors?.includes(prAuthor)) {
+    if (config.ignoreAuthors?.includes(sender)) {
```

Comment:
We may want to check both - consider the case where owlbot opens a PR, then I update the branch from main. I _think_ I would be the event `sender` (of `pull_request.synchronize`) but the PR may be legitimately updating a generated file.

Reply:
Good point - what are your thoughts on handling the use case where owl-bot opens a PR, then someone edits something that shouldn't be edited? Perhaps I should refactor to capture the files that changed between commits and create messages based on those files rather than all files in the PR itself

Reply:
That might work, but it's worth investigating what GitHub reports for files changed on a merge from main. The case I'm thinking of is a maintainer updates the branch from main and the files it pulls in might have updated a templated file (not sure what the data looks like). In that case, it might not be possible to distinguish whether or not to comment unless we special-case ignore the updating of a branch from the pull request base branch.

Reply:
Diving into this, there doesn't appear to be a clean way to get a clean diff between merges/commits. I can come back to work on this functionality if desired, while checking both the `prAuthor` and `sender` for now